steps/http/session.go

```diff
@@ -110,6 +110,12 @@ func (s *Session) ConfigureRequestBodyJSONProperties(ctx context.Context, props
 	return nil
 }
 
+// ConfigureRequestBodyJSONText sets a json string as the HTTP request's body .
+func (s *Session) ConfigureRequestBodyJSONText(ctx context.Context, message string) error {
+	s.Request.RequestBody = []byte(message)
```

Comment:
I guess that we should force the `Content-Type` HTTP header to `application/json` in both ConfigureRequestBodyJSONText and ConfigureRequestBodyJSONProperties.
Unless we want to add explicitly this HTTP header.
So, my proposal would be:
```
func (s *Session) ConfigureRequestBodyJSONText(ctx context.Context, message string) error {
	s.Request.RequestBody = []byte(message)
	if s.Request.Headers == nil {
		s.Request.Headers = make(map[string][]string)
	}
	s.Request.Headers["Content-Type"] = "application/json"
	return nil
}
```

and you can reuse this functionality from ConfigureRequestBodyJSONProperties to avoid repetition:

```
func (s *Session) ConfigureRequestBodyJSONProperties(ctx context.Context, props map[string]interface{}) error {
	var json string
	var err error
	for key, value := range props {
		if json, err = sjson.Set(json, key, value); err != nil {
			return fmt.Errorf("Error setting property '%s' with value '%s' in the request body. %s", key, value, err)
		}
	}
	return s. ConfigureRequestBodyJSONText(ctx, json)
}
```

Reply:
Good approach, implemented based in your example with a single change: include  s.Request.Headers["Content-Type"] = "application/json"  within the IF block

Reply:
Your proposal is wrong.
If you have a step with headers, then you will not include the content-type header.
Apart from that, append doesn't make sense (and is more verbose) because there is no headers. It wouldn't either make sense to add multiple Content-Type headers.
I notice a typo in my proposal:
```
	if s.Request.Headers == nil {
		s.Request.Headers = make(map[string][]string)
	}
	s.Request.Headers["Content-Type"] = []string{"application/json"}
```

Reply:
Understood the logic now:  Not a global constraint but related only to JSON steps
Done